screens/RoomListViewScreen/styles.js

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet';
+import { Colors } from '../../utils/colors';
+import { Typography } from '../../utils/typography';
+
+const styles = EStyleSheet.create({
+    img: {
+        backgroundColor: Colors.gray4,
+        flex: 3
```

Comment:
ang size sa figma do kay 137/186 ang size sa image

Reply:
To do